sdk/communication/Azure.Communication.Calling.Server/src/Models/Events/ParticipantsUpdatedEvent.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+
+namespace Azure.Communication.Calling.Server
+{
+    /// <summary>
+    /// The call state change event.
+    /// </summary>
+    public partial class ParticipantsUpdatedEvent : CallEventBase
+    {
+        /// <summary>
+        /// The event type.
+        /// </summary>
+        public const string EventType = "Microsoft.Communication.ParticipantsUpdated";
```

Comment:
Where is `Microsoft.Communication.ParticipantsUpdated` defined and why does it start by `Microsoft.Communication`?

Reply:
It's defined in PMA. That's why they are C# format namespaces.

Reply:
move to swagger , remove prefix of namespace